### ISSUE TITLE
Allow for non-pop login screen

### ISSUE THREAD

**Issue by ajmas** [dgrubelic/vue-authenticate#240]
Based on what I see in `options.js` and playing around with the `display` option, it does not current seem to be possible to force a non-pop-up authentication flow. This can be an issue for flows where a pop-up is not suitable.

**Comment by ajmas** [comment]
Since we lose the logic around the pop-up, maybe having an option to store state in session storage and then on page load have a `continueAuth()` call, which behind the scenes:
  
  - takes the page URL , as the pop up does
  - any state stored in session storage
  - the provider id

Maybe there is another approach?

**Comment by tomastan** [comment]
Agree. We have issues integrating this lib if the app is shown in restricted webview of a mobile app for example

@ajmas how did you resolve that back then?

**Comment by ajmas** [comment]
@tomastan I ended up [forking](https://github.com/ajmas/vue-authenticate-2) the project, but I'll need to look at my code to remind myself.

**Comment by ajmas** [comment]
@tomastan I just looked through the code and right now there isn't an alternative to the popup. Looks like some code rework is needed to provide that possibility. I'll create a ticket for this.

**Comment by tomastan** [comment]
> @tomastan I just looked through the code and right now there isn't an alternative to the popup. Looks like some code rework is needed to provide that possibility. I'll create a ticket for this.

Let me follow your fork then ;) Unfortunately as I understand it is Vue 3 only, while our codebase is not yet migrated. Plus if you said, popupless flow is not there, so likely we need to look for a temporary workaround.

**Comment by ajmas** [comment]
It is Vue 3 only. I made the choice because of our own project requirements and I couldn’t work out how to make it support both.
